NDTensors/src/lib/GradedAxes/src/gradedunitrange.jl

```diff
@@ -37,6 +37,11 @@ function Base.OrdinalRange{T,T}(a::GradedOneTo{<:LabelledInteger{T}}) where {T}
   return unlabel_blocks(a)
 end
 
+# == is just a range comparison that ignores labels. Need dedicated function to check equality.
+function gradedisequal(a1::AbstractUnitRange, a2::AbstractUnitRange)
+  return blockisequal(a1, a2) && (blocklabels(a1) == blocklabels(a2))
+end
```

Comment:
See comment above, probably it makes more sense to define:
```julia
function gradedisequal(a1::AbstractUnitRange, a2::AbstractUnitRange)
  return blockisequal(a1, a2)
end

function gradedisequal(a1::AbstractGradedUnitRange, a2::AbstractGradedUnitRange)
  return blockisequal(a1, a2) && (blocklabels(a1) == blocklabels(a2))
end
```
since `blocklabels` doesn't make sense for non-graded unit ranges.

Reply:
I plan to deal with this in #1531

Reply:
Could you explain how you will deal with it in #1531?